tools/make.py

```diff
@@ -264,6 +263,17 @@ def find_depbo_tools():
 
     return requiredToolPaths
 
+def pboproject_settings():
+    """Use registry entries to configure needed pboproject settings."""
+    value_exclude = "thumbs.db,*.txt,*.h,*.dep,*.cpp,*.bak,*.png,*.log,*.pew,source,*.tga"
```

Comment:
I seem to remember needing to remove *.h, but I don't think it's for ace, probably fine as it matches our wiki

Reply:
I should probably try this build and see if that causes any issues 🤔

Reply:
We don't use `.h` anywhere, always `.hpp`. :)

Reply:
`my/config.cpp: #includes \a3\editor_f\Data\Scripts\dikCodes.h, but is excluded from pbo`

was the error pboProject gives for my addon, but this doesn't effect ace
And it shouldn't even be an error....